mesh/profiles/views.py

```diff
@@ -1,3 +1,16 @@
-from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.http import JsonResponse
 
-# Create your views here.
+@csrf_exempt
```

Comment:
csrf exempt is still required because the cookie isnt properly configured in dev so views wont function without the flag. It can be removed in a future issue but we need this for now.